### ISSUE TITLE
Move Charter, process, contributing and other docs to this rep

### ISSUE THREAD

**Issue by wesleytodd** [expressjs/discussions#200]
Both https://github.com/expressjs/express/pull/5509 and https://github.com/expressjs/express/pull/5510 are great examples of why I think we should move this out of the express repo. We shouldn't need to run CI for these `.md` file changes. Also, it would be nice to add some markdown linting type stuff and keeping that out of this repo would be great. And thirdly, it would mean we don't have as many changes going into the autogenerated change logs for the main package.

Thoughts?

**Comment by jonchurch** [comment]
There is a piece in #170 which suggests doing this. I am in favor. 

Does OpenJSF have any guidance here? We could create a repo called `governance` to house that stuff. In fact, maybe this repo can be renamed to `governance` or `meta` as I can see this repo evolving in that direction.

**Comment by sheplu** [comment]
yes this would be great, but I would even suggest to create a dedicated repos for that (not sure about the name but discussion seems strange)
This was part of this issue I opened https://github.com/expressjs/discussions/issues/170 (you can find `Move all global document out of the express repository:` but also part of the expectations)

**Comment by wesleytodd** [comment]
I think the thing is moving it out of repos we ship libraries out of, so I am not opposed to a new repo but also since we have this one I also don't see a reason not to use this one.

**Comment by sheplu** [comment]
I was thinking of a more "admin" repo, I just created it for some PR about official records and decision (which also could be used to keep track of all minutes for the express TC meeting - and maybe all working groups). 
But no strong opinion - mostly to separate more administrative stuff and more global discussion

**Comment by UlisesGascon** [comment]
>  Also, it would be nice to add some markdown linting type stuff and keeping that out of this repo would be great.

My 2 cents on the linting matter. I find quite solid the linting workflow for markdown in place in the TODO Group as also reviews expressions and non-inclusive language:  https://github.com/todogroup/ospology/tree/main/.github/workflows

**Comment by jonchurch** [comment]
> Ok, so lets call that decided then and we are going to move all the governence and policy docs there. We likely need to keep some files in place to link from all the repos, but by removing the content we can reduce churn in all the repos for these kinds of changes.
> 
> https://github.com/expressjs/Admin

Since wes said this we have realized we already have a [`.github`](https://github.com/expressjs/.github) repo and were discussing in https://github.com/expressjs/Admin/pull/2 if these docs should live there instead.

Im up for it. If we do we need to decide if we want to delete the actual files like CONTRIBUTING.md from the other repos so the "defaults" commited to the `.github` would automatically get displayed on github or just leave the actual source with links pointing to the source docs. There's only a small benefit to deleting existing ones which are eligible to fallback to the default, which is the doc always being up to date. 

The specific files which are eligible to used as defaults this way in github are [listed in their docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types)

**Comment by wesleytodd** [comment]
Ah yeah I wasn't thinking about the shared thing with .github when discussing this. This is why I wish we had waited to create the new repo so we could gather this kind of feedback. The more I think about this the more I think the docs go in .github and the admin stuff goes in discussions. Idk, just important to keep things simple and have low duplication.

**Comment by wesleytodd** [comment]
I guess I didn't address the compelling reason to have more than one, I think the history in this discussions repo is the compelling reason. We don't want to make a new one because we don't want people to have to search multiple for convos. If we're going to do a different name that is all right, but we should move this repo not create a new one.

**Comment by jonchurch** [comment]
I introduced a third option about merging the `.github`, governance doc repo, and `disucssions` into a single repo to be used for all "meta" stuff. (it would be renaming discussions to .github so we dont lose history from discussions). Let's table that for now bc its distracting from the actual question...

### Do we want to have separate repos for `.github` and governance docs?

I vote no. We should use `.github` repo for Governance docs. Because it allows us to leverage the Defaults behavior/convention that comes w/ that.

**Comment by bjohansebas** [comment]
It has been several months since the last comment on this topic, and I would like to know if there is already consensus on this matter or if it hasn't been discussed in depth yet.

Part of my question is because these files are blocking https://github.com/expressjs/express/pull/6087.

**Comment by wesleytodd** [comment]
Good call adding to the agenda. We have been discussing some changes to this in the TC group, so we should for sure consolidate. There maybe be some outcomes as well from it for the content. We should make this a priority on the next meeting.

**Comment by bjohansebas** [comment]
I think we can move all this documentation to this repository, and with a script like the one we have on the website in the `.github` repository, include documents that can be replicated.

This repository is very frequently visited, so separating the information into `.github` might not be necessary.

**Comment by UlisesGascon** [comment]
I am attempted to work on this soon 😅

**Comment by bjohansebas** [comment]
We've already moved these documents out of the main repo. Now they're in discussions, and the ones that can be shared are in the .github repo. So I'm going to close this
